pyproject.toml

```diff
@@ -29,3 +35,14 @@ cache-keys = [
     { file = "python/scylla/**/*.pyi"},
 ]
 config-settings = { build-args = '--profile=dev' }
+
+[tool.ruff]
+line-length = 120
+
+[tool.basedpyright]
+typeCheckingMode = "strict"
+
+[tool.pytest.ini_options]
+markers = [
+    "smoke: smoke tests that require database",
+]
```

Comment:
I see that you still define some tests as tests that require database.
As discussed on the meeting, let's have a `requires_db` marker and put in on tests that need Scylla.